spec/spec.md

```diff
@@ -36,16 +36,20 @@ Credentials discoverable that belong to a DID or to a DNS name.
 
 Currently, the only way an application or web service can access VCs of a DID is to use a data exchange protocol like
 [OpenID4VC](https://openid.net/openid4vc/) or [DIDComm](https://didcomm.org/) to request and receive the desired
-credential. While this data exchange is important for VCs that shall be kept private, Holders might desire to make other
-VCs public. For public VCs the required data exchange increases the complexity of applications that want to discover and
-access these VCs because the application would need to implement a data exchange protocol.
+credential. While this data exchange is important for VCs that shall be kept private, [Holders][holder] might desire to
+make other VCs public. For public VCs the required data exchange increases the complexity of applications that want to
+discover and access these VCs because the application would need to implement a data exchange protocol.
 
 ## Potential Use Cases
 
 - Linking an official business registration credential (if there was such a thing) to a DNS name to reduce the burden of
   verifying the authenticity of a website.
 - Making the mandatory imprint page on a website machine readable by self-issuing an imprint credential and connecting
   it to the DID of the website to reduce the burden of vistors to extract relevant data from the imprint page.
+- for whom is this useful
+  - public entities
+  - long-lived identifiers
+- publicly proof ownership over a domain name - Well-known DID Configuration + published published credential
```

Comment:
I think this still needs some wordsmithing but this is a bit better
```suggestion
- public proof ownership over a domain name - Well-known DID Configuration + published credential
```

Reply:
Thanks, yes, I'm with you. This section is still raw. I created #21 to update it.